scripts/kind-e2e/lib_operator_verify_subm.sh

```diff
@@ -79,45 +79,35 @@ function verify_subm_cr() {
   kubectl get submariner --namespace=$subm_ns | grep $deployment_name
 
   # Show full SubM CR
-  kubectl get submariner $deployment_name --namespace=$subm_ns -o yaml
-
-  kubectl get submariner $deployment_name --namespace=$subm_ns -o jsonpath='{.metadata.namespace}' | grep $subm_ns
-  kubectl get submariner $deployment_name --namespace=$subm_ns -o jsonpath='{.apiVersion}' | grep submariner.io/v1alpha1
-  kubectl get submariner $deployment_name --namespace=$subm_ns -o jsonpath='{.kind}' | grep Submariner
-  kubectl get submariner $deployment_name --namespace=$subm_ns -o jsonpath='{.metadata.name}' | grep $deployment_name
-  kubectl get submariner $deployment_name --namespace=$subm_ns -o jsonpath='{..spec.brokerK8sApiServer}' | grep $SUBMARINER_BROKER_URL
-  kubectl get submariner $deployment_name --namespace=$subm_ns -o jsonpath='{.spec.brokerK8sApiServerToken}' | grep $SUBMARINER_BROKER_TOKEN
-  kubectl get submariner $deployment_name --namespace=$subm_ns -o jsonpath='{.spec.brokerK8sCA}' | grep $SUBMARINER_BROKER_CA
-  kubectl get submariner $deployment_name --namespace=$subm_ns -o jsonpath='{.spec.brokerK8sRemoteNamespace}' | grep $SUBMARINER_BROKER_NS
-  kubectl get submariner $deployment_name --namespace=$subm_ns -o jsonpath='{.spec.ceIPSecDebug}' | grep $ce_ipsec_debug
-  kubectl get submariner $deployment_name --namespace=$subm_ns -o jsonpath='{.spec.ceIPSecIKEPort}' | grep $ce_ipsec_ikeport
-  kubectl get submariner $deployment_name --namespace=$subm_ns -o jsonpath='{.spec.ceIPSecNATTPort}' | grep $ce_ipsec_nattport
+  json_file=/tmp/${deployment_name}.json
+  kubectl get submariner $deployment_name --namespace=$subm_ns -o json | tee $json_file
+
+  validate_equals '.metadata.namespace}' $subm_ns
```

Comment:
Shouldn’t these drop the closing bracket?

Reply:
right, keen eye! some bad vim-ing on my part :)